apps/asap-server/src/handlers/webhooks/webhook-run-migrations.ts

```diff
@@ -0,0 +1,257 @@
+  const getLatestMigrationPathFromDb = getLatestMigrationPathFromDbFactory(
+    client,
+  );
+  const getMigrationsFromPaths = getMigrationsFromPathsFactory(
+    importModule,
+    logger,
+  );
+  const removeExecutedMigration = removeExecutedMigrationFactory(client);
+
+  const migrationPath = await getLatestMigrationPathFromDb();
+
+  logger.debug({ migrations: migrationPath }, 'Latest migrations');
+
+  if (migrationPath !== null) {
+    let migration: Migration;
+    try {
+      [migration] = await getMigrationsFromPaths([migrationPath]);
+    } catch (error) {
+      logger.error(
+        error,
+        `Could not load the migration from file ${migrationPath}`,
+      );
+
+      return;
+    }
+
+    logger.debug(`Executing rollback of migration '${migration.getPath()}`);
+
+    try {
+      await migration.down();
+    } catch (error) {
+      logger.error(
+        error,
+        `Error executing the rollback of migration ${migration.getPath()}`,
+      );
+      return;
+    }
+
+    logger.debug('Finished executing rollback, saving progress...');
+
+    await removeExecutedMigration(migrationPath);
+
+    logger.info(`Rolled back and removed migration '${migration.getPath()}'`);
+  }
+};
+
+const getMigrationPathsFromDirectoryFactory = (
+  readDir: typeof fsPromise.readdir,
+) => async () => (await readDir(migrationDir)).sort();
+
+const getLatestMigrationPathFromDbFactory = (
+  client: Squidex<RestMigration>,
+) => async (): Promise<string | null> => {
+  const query: Query = {
+    take: 1,
+    skip: 0,
+    sort: [{ path: 'data.name.iv', order: 'descending' }],
+  };
+
+  const { items: migrations } = await client.fetch(query);
+
+  if (migrations.length === 0) {
+    return null;
+  }
+
+  return migrations[0].data.name.iv;
+};
+
+const importModuleFromPath = (path: string): Promise<Module> => import(path);
+export type ImportModuleFromPath = typeof importModuleFromPath;
+
+type Module = {
+  default?: { new (path: string): Migration | unknown };
+};
+
+const getMigrationsFromPathsFactory = (
+  importModule: typeof importModuleFromPath,
+  logger: Logger,
+) => async (migrationPaths: string[]): Promise<Migration[]> => {
+  const migrations = Promise.all(
+    migrationPaths.map(async (file) => {
+      logger.debug({ file });
+      const { default: ImportedModule } = await importModule(
+        `${migrationDir}/${file}`,
+      );
+
+      if (typeof ImportedModule !== 'function') {
+        throw new Error(`${file} does not export a valid module`);
+      }
+
+      const migration = new ImportedModule(file);
+
+      if (!isMigration(migration)) {
+        throw new Error(`${file} does not contain a valid migration`);
+      }
+
+      return migration;
+    }),
+  );
+
+  return migrations;
+};
+
+const filterUnexecutedMigrationsFactory = (client: Squidex<RestMigration>) =>
+  filterMigrationsFactory(client);
+
+const filterMigrationsFactory = (
+  client: Squidex<RestMigration>,
+  unexecuted = true,
```

Comment:
```suggestion
  executed = false,
```
To prevent double negation, I think it's easier to read :)